src/components/TopBar/TopBar.js

```diff
@@ -1,15 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { createPortal } from 'react-dom';
 import Logo from '../Logo/Logo';
+import Modal from '../../modals/Modal/Modal';
+import AddMovieModal from '../../modals/AddMovieModal/AddMovieModal';
 import './TopBar.scss';
 
 export default function TopBar() {
-  const addMovie = () => {
-  };
+  const [showModal, setShowModal] = useState(false);
 
   return (
     <div className='top-bar'>
       <Logo />
-      <button type="button" onClick={addMovie}>+ ADD MOVIE</button>
+      <button
+        type="button"
+        className='add-movie__button'
+        onClick={() => setShowModal(true)}
+      >
+        + ADD MOVIE
+      </button>
+      {showModal && createPortal(
```

Comment:
Used `createPortal()` only here  as a temporary solution just for example.
I don't really understand the benefits of this.